python/coinbase-agentkit/pyproject.toml

```diff
@@ -21,7 +21,7 @@ allora-sdk = "^0.2.0"
 paramiko = "^3.5.1"
 nilql = "^0.0.0a12"
 ecdsa = "^0.19.0"
-pyjwt = {extras = ["crypto"], version = "^2.10.1"}
+pyjwt = {extras = ["crypto"], version = "0.3.0"}
```

Comment:
@0xRAG I'll manually revert this, but there is a flaw. We probably want the parse to make sure the character before `version` is a new-line